### DISCUSSION TITLE
Add post to beginning of queue

### DISCUSSION THREAD

**Discussion by Vickysaurus** [AprilSylph/XKit-Rewritten#1072] [Suggestion Box]
Is it possible to add a function to add a post to the beginning of the queue instead of its end? Sometimes I want a post to show up reasonably quickly but still want to avoid posting a whole bunch of stuff all at once and currently the way to accomplish this is queue the post -> go to queue -> scroll all the way down past the other 270 post in the queue -> push the button to jump it to the beginning of the queue. Which is a little cumbersome even with collapsed queue.

**Comment by arcanetrivia** [comment]
I use the Scroll To Bottom script to make the scrolling itself less painful on those times I have a long queue, but I've sometimes wished for this as well, for the same use case (I want this post to be next, but not immediate).

In the same vein, I wonder if it would be possible to do the reverse with drafts, and have a new draft save at the bottom rather than the top? When I put things into queue from drafts, I almost always want to do so starting from the oldest draft; which if I have any significant number, there's the same thing of having to scroll all the way to the bottom first to find it (although again the Scroll to Bottom script is handy for that).

**Comment by marcustyphoon** [comment]
Definitely suggest this to to [tumblr.com/support](https://tumblr.com/support)! I would love this as a built-in feature of the post editor.
